app/assets/stylesheets/style.css.scss

```diff
@@ -131,14 +178,37 @@ fieldset.tagline {
   font-family: Lato;
 }
 
-#neighborhood h1 {
-  font-family: Lato;
+#neighborhood .one_neighborhood {
+  width: 100%;
 }
 
-#neighborhood_info {
-  padding-top: 4em;
+#neighborhood h1 {
+  font-family: Lato;
 }
 
 #ratings_list {
   padding-top: 3em;
 }
+
+#contributors {
+  border: #B23875 solid .2em;
+  border-radius: 1em;
+  margin-top: 4em;
+  margin-bottom: 4em;
+  height: 20em;
+  background-image: url("https://www.georgeglazer.com/maps/pictorialmap/images/gar-bostondet.JPG");
+}
+
+#contributors p {
+  margin-left: 1em;
+  margin-top: 2em;
+  font-family: Lato;
+  color: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.